features/mytesting.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Test feature
```

Comment:
Test scenarios should goes to `features/test/`

Reply:
Removed it. Need not add it as test scenario. Only for quicker test for pass log.